test/protocol/SequentialCommitHandlerTest.js

```diff
@@ -592,7 +594,37 @@ describe("IBosonSequentialCommitHandler", function () {
                 sequentialCommitHandler
                   .connect(buyer2)
                   .sequentialCommitToOffer(buyer2.address, tokenId, priceDiscovery, { value: price2 })
-              ).to.revertedWithCustomError(bosonErrors, RevertReasons.REGION_PAUSED);
+              )
+                .to.revertedWithCustomError(bosonErrors, RevertReasons.REGION_PAUSED)
+                .withArgs(PausableRegion.Buyers);
+            });
+
+            it("The sequential region of protocol is paused", async function () {
+              // Pause the sequential commit region of the protocol
+              await pauseHandler.connect(pauser).pause([PausableRegion.SequentialCommit]);
+
+              // Attempt to sequentially commit, expecting revert
+              await expect(
+                sequentialCommitHandler
+                  .connect(buyer2)
+                  .sequentialCommitToOffer(buyer2.address, tokenId, priceDiscovery, { value: price2 })
+              )
+                .to.revertedWithCustomError(bosonErrors, RevertReasons.REGION_PAUSED)
+                .withArgs(PausableRegion.SequentialCommit);
+            });
+
```

Comment:
Could be interesting to test that commit to a fixed price offer and commit to a price discovery offer are still possible, even when the SequentialCommit region is Paused

Reply:
Added in appropriate files (exchangeHandlerTests and priceDiscoveryTest)